canonical-web-and-design/deployment-configs
# Redirect blog to ubuntu.com

This is the next step in releasing ubuntu.com/blog. We will redirect blog.ubuntu.com/* to ubuntu.com/blog/*.

QA
--

``` bash
./qa-deploy --production ubuntu.com --staging staging.ubuntu.com --tag latest
watch microk8s.kubectl get all,ingress
```

Once the service is "Running" and the ingresses have IP addresses assigned, then do:

``` bash
$ curl -I -H 'Host: blog.ubuntu.com' --insecure https://127.0.0.1
$ curl -I -H 'Host: blog.ubuntu.com' --insecure https://127.0.0.1/subpath?query
$ curl -I -H 'Host: blog.staging.ubuntu.com' --insecure https://127.0.0.1
$ curl -I -H 'Host: blog.staging.ubuntu.com' --insecure https://127.0.0.1/subpath?query
```

In all cases, check you see a 301 response whose `Location` is ubuntu.com/blog and staging.ubuntu.com/blog, respectively.
